x/callback/keeper/callback.go

```
@@ -27,18 +27,26 @@ func (k Keeper) GetCallbacksByHeight(ctx sdk.Context, height int64) (callbacks [
 }
 
 // ExistsCallback returns true if the callback exists for height with same contract address and same job id
-func (k Keeper) ExistsCallback(ctx sdk.Context, height int64, contractAddress sdk.AccAddress, jobID uint64) (bool, error) {
+func (k Keeper) ExistsCallback(ctx sdk.Context, height int64, contractAddr string, jobID uint64) (bool, error) {
+	contractAddress := sdk.MustAccAddressFromBech32(contractAddr)
 	return k.Callbacks.Has(ctx, collections.Join3(height, contractAddress.Bytes(), jobID))
 }
 
+// GetCallback returns the callback given the height, contract address and job id
+func (k Keeper) GetCallback(ctx sdk.Context, height int64, contractAddr string, jobID uint64) (types.Callback, error) {
+	contractAddress := sdk.MustAccAddressFromBech32(contractAddr)
+	return k.Callbacks.Get(ctx, collections.Join3(height, contractAddress.Bytes(), jobID))
+}
+
 // DeleteCallback deletes a callback given the height, contract address and job id
-func (k Keeper) DeleteCallback(ctx sdk.Context, sender string, height int64, contractAddress sdk.AccAddress, jobID uint64) error {
+func (k Keeper) DeleteCallback(ctx sdk.Context, sender string, height int64, contractAddr string, jobID uint64) error {
+	contractAddress := sdk.MustAccAddressFromBech32(contractAddr)
 	// If callback delete is requested by someone who is not authorized, return error
```

let's use the `sdk.AccAddressFromBech32`  since we can return an error here.